corehq/apps/reminders/dbaccessors.py

```diff
@@ -0,0 +1,9 @@
+def get_surveys_in_domain(domain):
+    from corehq.apps.reminders.models import Survey
+    return sorted(Survey.view(
```

Comment:
shouldn't these already be sorted given that the domain is the same, survey should be in lexicographical order? though i suppose this might be emitting the id and not the name